ui/shared/Input.qml

```diff
@@ -70,6 +71,7 @@ Item {
             background: Rectangle {
                 color: "#00000000"
             }
+            Keys.onReleased: onKeyReleased()
```

Comment:
You probably can access that using `textField.Keys.onReleased`. I don't think we need to add this new prop

Reply:
cool! I'll try that. I did not know that was possible!